popup.js

```diff
@@ -0,0 +1,195 @@
+      sourceCode: 'https://github.com/ashmalzahra/Portfolio',
+    },
+    index: 0,
+    class: 'project',
+  },
+];
+
+const articlesContainer = document.createElement('section');
+articlesContainer.classList.add('work');
+articlesContainer.id = 'portfolio';
+
+let text = '';
+
+projects.forEach((element) => {
+  if (element === projects[0]) {
+    text += `
+    <h3 class="${projects[0].class1}">${projects[0].title1}</h3>
+    <hr id="line">
+
+    <img class="main-project" src="${projects[0].img1}" alt="GymFit">
+    <img class="desktop-project" src="${projects[0].img2}" alt="GymFit">
+
+    <article class="${projects[0].class}">
+    <h2>${projects[0].title}</h2>
+    <p>${projects[0].description}</p>
+    <ul id="languages">
+    <li>${projects[0].technologies[0]}</li>
+    <li>${projects[0].technologies[1]}</li>
+    <li>${projects[0].technologies[2]}</li>
+    <li>${projects[0].technologies[3]}</li>
+    </ul>
+    <button class="see-project">See Project</button>
+    </article>`;
+  } else if ('title2' in element) {
+    text += `
+    <article class="${element.class}" id="${element.id}">
+    <h2 class="mobile-version">${element.title}</h2>
+    <h2 class="desktop-version">${element.title2}</h2>
+    <p>${element.description}</p>
+    <ul class="languages">
+    <li>${element.technologies[0]}</li>
+    <li>${element.technologies[1]}</li>
+    <li>${element.technologies[2]}</li>
+    </ul>
+    <button class="button">See Project</button>
+    </article>
+   `;
+  } else {
+    text += `
+      <article class="${element.class}" id="${element.id}">
+      <h2>${element.title}</h2>
+      <p>${element.description}</p>
+      <ul class="languages">
+      <li>${element.technologies[0]}</li>
+      <li>${element.technologies[1]}</li>
+      <li>${element.technologies[2]}</li>
+      </ul>
+      <button class="button">See Project</button>
+      </article>
+  `;
+  }
+});
+
+articlesContainer.innerHTML = text;
+
+document.body.insertBefore(articlesContainer, document.body.children[2]);
+
+const closebutton = document.querySelector('.popup #close');
+
+const projectWindow = document.querySelector('.popup');
+
+// const Title = document.getElementById('popupH2');
+// const Tech = document.querySelector('.popup-list');
+// // const imgM = document.getElementById('imgM');
+// // const imgD = document.getElementById('imgD');
+// const Description = document.querySelector('.popup-description');
+// const Live = document.getElementById('Live');
+// const Source = document.getElementById('Source');
+
+// const createPopup = (projects) => {
+//   Title.textContent = projects.title;
+//   Description.textContent = projects.description;
+//   Tech.innerHTML = projects.technologies.map((technology) => `<li>${technology}</li>`).join('');
+//   Live.forEach((link) => {
+//     link.href = projects.links.liveVersion;
+//   });
+//   Source.forEach((link) => {
+//     link.href = projects.links.sourceCode;
+//   });
+// };
```

Comment:
- [ ] [OPTIONAL] I highly suggest you remove unused commented code from the codebase to make our work more clean and professional. 😎️